manifests/init.pp

```diff
@@ -54,20 +54,24 @@
   $purge_inputs   = false,
   $purge_outputs  = false,
 ) inherits splunk::params {
-  include staging
 
   $virtual_service = $splunk::params::server_service
-  $staged_package  = staging_parse($package_source)
   $staging_subdir  = $splunk::params::staging_subdir
 
   $path_delimiter  = $splunk::params::path_delimiter
-  $pkg_path_parts  = [$staging::path, $staging_subdir, $staged_package]
-  $pkg_source      = join($pkg_path_parts, $path_delimiter)
 
-  staging::file { $staged_package:
-    source => $package_source,
-    subdir => $staging_subdir,
-    before => Package[$package_name],
+  if $pkg_provider != undef and $pkg_provider != 'yum' and  $pkg_provider != 'apt' {
```

Comment:
Should we just go ahead and change this to `if $pkg_provider == 'rpm' or $pkg_provider == 'deb'` ? I wonder if anyone uses this on windows too, since it has some windows stuff in the module. Perhaps the msi provider?